cloudwatch/cloudwatch.go

```diff
@@ -0,0 +1,105 @@
+// Package cloudwatch defines structures for interacting with Cloudwatch Metrics.
+package cloudwatch // import "bosun.org/cloudwatch"
```

Comment:
I'm wondering if there is a better place for this package than off of the root? Perhaps cmd/bosun/cloudwatch

Reply:
I'd based it off the graphite backend, so was copying the pattern where there was an top level graphite and opentsdb folder with the struct representing the query objects.  I notice that the newer data sources like elastic and influx are in `cmd/bosun/expr/` 
I could refactor it to just all live in the `cmd/bosun/expr/cloudwatch.go` file if that works for you ?